manifest.json

```diff
@@ -20,6 +20,7 @@
   },
   "dependencies": {
     "vtex.product-summary": "1.x",
-    "vtex.store-components": "2.x"
+    "vtex.store-components": "2.x",
+    "vtex.product-details": "0.8.1"
```

Comment:
Why product-kit depends on product-details? 🤔

Reply:
I'm re-using the propTypes validation of ProductDetails.